assets/javascripts/modules/analytics/awin.js

```diff
@@ -11,14 +11,15 @@ define([
         AWIN.Tracking.Sale = {};
 
         /*** Set your transaction parameters ***/
-        const productData = guardian.pageInfo.productData
+        var productData = guardian.pageInfo.productData
```

Comment:
Why did you need to change this to a `var`?

Reply:
I wasn't sure which js version we were targeting and Intellij was complaining. Digging a bit deeper though, it's fine so I've reinstated it.